services/video_indexer/actual_board_detection.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+
+def imageRowToBlackWhiteCountArray(image_row):
+    white_count = 0
+    black_count = 0
+
+    black_white_count_array = np.array([],dtype=int)
+
+    for i in image_row:
+        if i == 0:
+            if white_count != 0:
+                black_white_count_array = np.append(black_white_count_array, white_count)
+                white_count = 0
+            black_count += 1    
+        else:
+            if black_count != 0:
+                black_white_count_array = np.append(black_white_count_array,black_count)
+                black_count = 0
+            white_count += 1
+
+    if white_count != 0:
```

Comment:
What's with this extra check of white_count outside the loop above? After it's looped through each of the pixels in the row the last one isn't handled?